src/Symfony/Component/DependencyInjection/Tests/Loader/PhpFileLoaderTest.php

```diff
@@ -61,7 +61,7 @@ public function testPrependExtensionConfigWithLoadMethod()
         $container = new ContainerBuilder();
         $container->registerExtension(new \AcmeExtension());
         $container->prependExtensionConfig('acme', ['foo' => 'bar']);
-        $loader = new PhpFileLoader($container, new FileLocator(\dirname(__DIR__).'/Fixtures/config'), 'prod', new ConfigBuilderGenerator(sys_get_temp_dir()), true);
+        $loader = new PhpFileLoader($container, new FileLocator(\dirname(__DIR__).'/Fixtures/config'), 'prod', prepend: true);
```

Comment:
Tests should not use named parameters. This hides unintentional BC breaks that would be detected due to tests being forced to be updated, by making tests used the public API.